### PR TITLE
Payfinecontrol dipanshu

### PR DESCRIPTION
**Pull request by dipanshu42**
Hey @navjeevan24  as my mediator can you please tell @dharam4  to review my payfinecontrol class as i have corrected the code according to the style guideline. so please ask him to give his feedback.

### CODE REVIEW

File: library/Main.java [hunk 1]
```diff
@@ -53,7 +54,27 @@ private static String Get_menu() {
 		  .append("  T  : increment date\n")
 		  .append("  Q  : quit\n")
 		  .append("\n")
-		  .append("Choice : ");
+		  .append("Choice : ");*/
+		
+		  sb.append("  M  : add member\n")
```
Comment (dharam4) [inline library/Main.java:56]: If you wanna pout sb.append then there would be semicolon in the end of each line.

File: library/Main.java [hunk 2]
```diff
@@ -82,10 +82,10 @@ private static String getMenu()               //private static String Get_menu()
 
 	public static void main(String[] args) {		
 		try {			
-			IN = new Scanner(System.in);
-			LIB = Library.GeTiNsTaNcE();
-			CAL = Calendar.gEtInStAnCe();
-			SDF = new SimpleDateFormat("dd/MM/yyyy");
+			in = new Scanner(System.in);                 //IN = new Scanner(System.in);
+			lib = Library.GeTiNsTaNcE();                 //LIB = Library.GeTiNsTaNcE();
```
Comment (dharam4) [inline library/Main.java:83]: GeTiNsTaNcE should be getInstace

File: library/Main.java [hunk 3]
```diff
@@ -82,10 +82,10 @@ private static String getMenu()               //private static String Get_menu()
 
 	public static void main(String[] args) {		
 		try {			
-			IN = new Scanner(System.in);
-			LIB = Library.GeTiNsTaNcE();
-			CAL = Calendar.gEtInStAnCe();
-			SDF = new SimpleDateFormat("dd/MM/yyyy");
+			in = new Scanner(System.in);                 //IN = new Scanner(System.in);
+			lib = Library.GeTiNsTaNcE();                 //LIB = Library.GeTiNsTaNcE();
+			CAL = Calendar.gEtInStAnCe();                //CAL = Calendar.gEtInStAnCe();
```
Comment (dharam4) [inline library/Main.java]: GeTiNsTaNcE should be corrected

File: library/Main.java [hunk 4]
```diff
@@ -84,8 +84,8 @@ public static void main(String[] args) {
 		try {			
 			in = new Scanner(System.in);                 //IN = new Scanner(System.in);
 			lib = Library.GeTiNsTaNcE();                 //LIB = Library.GeTiNsTaNcE();
-			CAL = Calendar.gEtInStAnCe();                //CAL = Calendar.gEtInStAnCe();                
-			SDF = new SimpleDateFormat("dd/MM/yyyy");     //SDF = new SimpleDateFormat("dd/MM/yyyy");
+			cal = Calendar.gEtInStAnCe();                //CAL = Calendar.gEtInStAnCe();
```
Comment (dharam4) [inline library/Main.java:84]: here also

**Review by dharam4 — CHANGES_REQUESTED** [review]
hi dipanshu , there are some errors which i mentioned in the comments please correct these as soon as possible

**Comment by dipanshu42** [comment]
Sure i will check and let you know

**Comment by dipanshu42** [comment]
Hey @navjeevan24  i have uploaded the code as per the suggested work, so now please ask the reviewer @dharam4  to go through it again and provide me his feedback.

**Review by dharam4 — APPROVED** [review]
yes all good @navjeevan24  , you can send the merge approval to dipanshu.

**Pull request merged**